code/codegen/src/main/kotlin/com/android/gradle/replicator/resgen/util/XmlImageGenerator.kt

```diff
@@ -0,0 +1,4 @@
+package com.android.gradle.replicator.resgen.util
+
+class XmlImageGenerator {
```

Comment:
what is this ? comments ?

Reply:
This was the old name for VectorDrawableGenerator. Git sometimes gets confused and re-adds files that were deleted.